pkg/mount-manager/fake-safe-mounter.go

```diff
@@ -52,35 +52,24 @@ func NewFakeSafeMounter() *mount.SafeFormatAndMount {
 
 type FakeBlockingMounter struct {
 	*mount.FakeMounter
-	mountToRun   chan MountSourceAndTarget
-	readyToMount chan struct{}
+	ReadyToExecute chan chan struct{}
 }
 
-type MountSourceAndTarget struct {
-	Source string
-	Target string
-}
-
-// FakeBlockingMounter's method adds two channels to the Mount process in order to provide functionality to finely
-// control the order of execution of Mount calls. readToMount signals that a Mount operation has been called.
-// Then it cycles through the mountToRun channel, waiting for permission to actually make the mount operation.
+// FakeBlockingMounter's method adds functionality to finely control the order of execution of Mount calls.
```

Comment:
nit: Godoc should begin with name of the element it describes (in this case `Mount`)